packages/framer-motion/src/animation/optimized-appear/start.ts

```diff
@@ -24,40 +39,43 @@ export function startOptimizedAppearAnimation(
 
     const storeId = appearStoreId(id, name)
 
-    /**
-     * Use a dummy animation to detect when Chrome is ready to start
-     * painting the page and hold off from triggering the real animation
-     * until then.
-     *
-     * https://bugs.chromium.org/p/chromium/issues/detail?id=1406850
-     */
-    const readyAnimation = animateStyle(
-        element,
-        name,
-        [keyframes[0] as number, keyframes[0] as number],
-        /**
-         * 10 secs is basically just a super-safe duration to give Chrome
-         * long enough to get the animation ready.
-         */
-        { duration: 10000, ease: "linear" }
-    )
+    if (!readyAnimation) {
+        readyAnimation = animateStyle(
+            element,
+            name,
+            [keyframes[0] as number, keyframes[0] as number],
+            /**
+             * 10 secs is basically just a super-safe duration to give Chrome
+             * long enough to get the animation ready.
+             */
+            { duration: 10000, ease: "linear" }
+        )
 
-    appearAnimationStore.set(storeId, {
-        animation: readyAnimation,
-        startTime: null,
-    })
+        appearAnimationStore.set(storeId, {
+            animation: readyAnimation,
+            startTime: null,
+        })
```

Comment:
Not introduced in this PR, but why do we need to store the dummy animation in the appearAnimationStore?